### ISSUE TITLE
SVC depends on kotlin-reflect

### ISSUE THREAD

**Issue by 1951FDG** [BansookNam/svc-lotto#1]
Hi,

Since there is no Issues tab for repo: https://github.com/BansookNam/svc, I'm asking here.

Does SVC require kotlin-reflect dependency? I am using SVC in one of my apps, but the kotlin-reflect dependency is quite big.

**Comment by BansookNam** [comment]
Hi, @1951FDG 
kotlin-reflect is not required to use.
what version of SVC are you using?

if you use 
```
implementation "com.naver.android.svc:svc:1.0.0"
```

I'm sure you don't need  kotlin-reflect dependency.

This lotto example also not need the dependency.

I'm not sure if the answer is correct.
If you have problem still give comment below.

And I'll open issue tab there.

Thank you for notice!

**Comment by 1951FDG** [comment]
Hi, @BansookNam 

Indeed, I've excluded the dependency as a workaround:
```
implementation('com.naver.android.svc:svc:1.0.0') {
        exclude group: 'org.jetbrains.kotlin', module: 'kotlin-reflect'
    }
```
It does however by default include the kotlin-reflect dependency, if you look at:
https://github.com/BansookNam/svc/blob/master/svc/build.gradle

**Comment by BansookNam** [comment]
Oh, you are right. I'll check it out if this is necessary.

**Comment by BansookNam** [comment]
@1951FDG I've just published 1.0.1 please check if it is solved.

**Comment by 1951FDG** [comment]
@BansookNam Perfect, tested, works!
